website/components/mega-nav/index.jsx

```diff
@@ -0,0 +1,16 @@
+import InlineSvg from '@hashicorp/react-inline-svg'
+import hashicorpLogo from './img/hashicorp-logo.svg?include'
+import TemporarySuite from './temporary_suite'
+
+export default function MegaNav({ product }) {
```

Comment:
Can we use the existing `mega-nav` component from `web-components`. Vault is [using it](https://github.com/hashicorp/vault/blob/f4dcdb01493288f9f0c22b84a7e1179287392dd8/website/package.json#L21)

Reply:
So, this is another copy-paste largely for expediency purposes.  This is definitely debt, that will need to be addressed, but it will need to be carefully handled as we roll out the new subnav between different sites.  This is another topic of conversation for our web-components meeting.

In short, the new subnav design clashes with the mega-nav design at large screen sizes.  We basically wanted to ensure the meganav & subnav elements lined up.

<img width="2210" alt="Screen Shot 2020-02-23 at 1 16 15 PM" src="https://user-images.githubusercontent.com/2105067/75120275-375d2100-563f-11ea-88cf-c8805c651b23.png">
<img width="2451" alt="Screen Shot 2020-02-23 at 1 16 26 PM" src="https://user-images.githubusercontent.com/2105067/75120276-37f5b780-563f-11ea-98ec-9d2897ab22a4.png">